source/github-helpers/index.ts

```diff
@@ -37,6 +37,34 @@ export const getCurrentCommittish = (pathname = location.pathname, title = docum
 	return unslashedCommittish;
 };
 
+export const getCurrentBranchFromFeedLink = (): string | undefined => {
+	// .last needed for #2799
+	const feedLink = select.last('link[type="application/atom+xml"]');
```

Comment:
If you have time, let's verify that #2799 still applies and if not let's add a // GHE #4187